.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -97,6 +97,16 @@ configure_namespace() {
   oc config set-context --current --namespace="${project}"
 }
 
+configure_namespace_if_nonexistent() {
```

Comment:
@Omar-AlJaljuli I don't see this method being used. can you please remove?

Reply:
@Omar-AlJaljuli  Why it was marked as resolved since this code is still here and not being used?